### PR TITLE
fix(color-picker): Separate popup from color picker

### PR DESCRIPTION
**Pull request by mannycarrera4**
## Summary

Fixes: https://github.com/Workday/canvas-kit/issues/836

## BREAKING CHANGE
• Remove the popup from color picker
• Color picker now renders just the swatches
• If you want to render the color picker in a popup, you must wrap it in a popup component

### CODE REVIEW

File: modules/_labs/color-picker/react/lib/ColorPicker.tsx [hunk 1]
```diff
@@ -94,6 +94,10 @@ const defaultColors = [
   colors.blackPepper400,
 ];
 
+const ColorPickerContainer = styled('div')({
+  width: 216,
```
Comment (mannycarrera4) [inline modules/_labs/color-picker/react/lib/ColorPicker.tsx:98]: Not sure how we want to handle this, at this set width the columns align perfectly, should we make this a prop or add an explanation in the code as to why it;s this set width?

File: modules/_labs/color-picker/react/stories/stories_ColorPicker.tsx [hunk 2]
```diff
@@ -5,7 +5,8 @@ import {action} from '@storybook/addon-actions';
 import withReadme from 'storybook-readme/with-readme';
 import {ColorInput} from '@workday/canvas-kit-react-color-picker';
 import {colors} from '@workday/canvas-kit-react-core';
-import {Popper} from '@workday/canvas-kit-react-popup';
+import {TransformOrigin, getTranslateFromOrigin} from '@workday/canvas-kit-react-common';
```
Comment (mannycarrera4) [inline modules/_labs/color-picker/react/stories/stories_ColorPicker.tsx:8]: remove

**Comment by NicholasBoll** [comment]
How should we message this change? It would be a breaking change to anyone using the popup pattern.

**Review by NicholasBoll — APPROVED** [review]

**Comment by mannycarrera4** [comment]
Should i add something to the readme and more detail to the Pull request info?

**Comment by NicholasBoll** [comment]
We haven't really broken labs much. I think a warning in the pull request description for sure. We'll want to add something to the squash commit message. We don't have an SLA of semantic versioning for labs, but we can still add breaking change messages. People would appreciate the heads-up.

**Pull request merged**